matRad_dijSampler.m

```diff
@@ -0,0 +1,63 @@
+function [cst,dij] = matRad_dijSampler(cst,ct,dij, margin_size,sparCity)
+% marginsize given in pixel thickness
+% fraction of voxels to be sampled [ Target OAR]
+
+m = (2*margin_size)+1;
+msk = ones(m,m,m);    % mask for convolution
+
+volThresh = 1000;  % setting full sampling for small structures 
```

Comment:
better comment: % stuctures where the number of voxels is smaller that volThres will be ommitted from sampling.